Sources/Host/Linux/X86_64/PTraceX86_64.cpp

```diff
@@ -42,21 +42,22 @@ void PTrace::initCPUState(ProcessId pid) {
 
 void PTrace::doneCPUState() { delete _privateData; }
 
-static inline void user_to_state32(ds2::Architecture::X86_64::CPUState32 &state,
-                                   user_fpregs_struct const &user) {
+static inline void
+xfpregs_to_state32(ds2::Architecture::X86_64::CPUState32 &state,
```

Comment:
You can keep the old name, `user_to_state32`. I think we just use "user" everywhere to describe the structure that comes from the thread's context.